superset-frontend/src/dashboard/components/DrillDetailPane/DrillDetailPane.tsx

```diff
@@ -242,8 +267,77 @@ export default function DrillDetailPane({
     );
   }
 
+  // Get datasource metadata
+  const response = useApiV1Resource<Dataset>(`/api/v1/dataset/${datasourceId}`);
+
+  const metadata = useMemo(() => {
+    const { status, result } = response;
+    const items: ContentType[] = [];
+    if (result) {
+      const {
+        changed_on,
+        created_on,
+        description,
+        table_name,
+        changed_by,
+        created_by,
+        owners,
+      } = result;
+      const notAvailable = t('Not available');
+      const createdBy = created_by
+        ? `${created_by.first_name} ${created_by.last_name}`
+        : notAvailable;
```

Comment:
I don't know whether the `first_name` and `last_name` would have been null values, so get rid of `undefined or null` value in the object might make sense.

```javascript
       const createdBy = `${created_by?.first_name ?? ""} ${created_by?.last_name ?? ""}`.trim() || notAvailable;
```

Reply:
Nice catch! Done.